FileSyncandWOPI/Source/MS-FSSHTTP-FSSHTTPB/TestSuite/MS-FSSHTTP-FSSHTTPB_SharePointFoundation2010_SHOULDMAY.deployment.ptfconfig

```diff
@@ -227,5 +227,7 @@
     <Property name="R246801Enabled_MS-FSSHTTP-FSSHTTPB" value="false"/>
     <!--Set R1348Enabled_MS-FSSHTTP-FSSHTTPB to true to indicate that the implementation does support AllowFragments2 field. R1348Enabled_MS-FSSHTTP-FSSHTTPB to false to disable this requirement. -->
     <Property name="R1348Enabled_MS-FSSHTTP-FSSHTTPB" value="false"/>
+    <!--Set R1360Enabled_MS-FSSHTTP-FSSHTTPB to false to indicate that the implementation does not support the Data Hash Header. R1360Enabled_MS-FSSHTTP-FSSHTTPB to true to disable this requirement. -->
```

Comment:
Update to "Set R1360Enabled_MS-FSSHTTP-FSSHTTPB to true to indicate that the implementation does support the Data Hash Header. R1360Enabled_MS-FSSHTTP-FSSHTTPB to false to disable this requirement."  this comment in all ptfconfig file should same.